quandl/api_config.py

```diff
@@ -1,9 +1,16 @@
 class ApiConfig:
     api_key = None
-    api_base = 'https://www.quandl.com/api/v3'
+    api_protocol = 'https://'
+    api_base = '{}www.quandl.com/api/v3'.format(api_protocol)
     api_version = None
     page_limit = 100
 
+    use_retries = True
+    number_of_retries = 8
+    retry_backoff_factor = 0.1
+    max_wait_between_retries = 15
```

Comment:
i wonder if they agree with this 15 rule? it's not in the ticket 😹

Reply:
I went over these defaults with Pegah who said they were reasonable. Its open to discussion though

Reply:
@ericvautour-quandl Is there a value you'd suggest? If we bump the retry_backoff_factor I'd be in favour of reducing either the max_wait_between_retries or the number of retries